src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -289,6 +292,9 @@ public class KNNSettings {
         }
     };
 
+    private final static Map<String, Setting<?>> FEATURE_FLAGS = KNNFeatureFlags.getFeatureFlags()
```

Comment:
I guess u forgot to remove KNNFeatureFlags here , it's just like getFeatureFlags(), since it's already added in import

Reply:
I think if we want imports a certain way we should try and encapsulate in checkstyle, its going to be hard to keep a track of it.